techniques/general/G219.html

```diff
@@ -20,8 +20,7 @@ <h2>Applicability</h2>
 			<h2>Description</h2>
 			<p>The objective of this Technique is to ensure that people with motor impairments who cannot carry out <a>dragging movements</a> are presented with an alternative <a>single pointer</a> interaction.</p> 
       <p>Some direct manipulation interfaces allow users to pick up targets and use dragging movements to move them to another position, for example, to change the position of an item in a priority list, or to move a task on a Kanban or planning board.</p>
-      <p>Such dragging movements are difficult or impossible to carry out for some users with motor disabilities. The alternative to dragging movements operates the underlying function by one or several <a>single pointer</a> activations. A single tap or click may reveal controls (arrows) to move a target in a stepwise fashion; open a drop-down menu where the drop position can be selected; or allow moving it to an ajacent postion by a swipe gesture.</p>
-      
+      <p>Such dragging movements are difficult or impossible to carry out for some users with motor disabilities. The alternative to dragging movements operates the underlying function by one or several <a>single pointer</a> activations. A single tap or click may reveal controls (arrows) to move a target in a stepwise fashion; open a drop-down menu where the drop position can be selected; or allow moving it to an ajacent postion by a swipe gesture. (Regarding the swipe gesture alternative, note that the swipe gesture alone would not be sufficient since Success Criterion 2.5.1 Pointer Gestures requires that functionality can operated with a single pointer <strong>without</strong> a path-based gesture, unless a multipoint or path-based gesture is essential.)</p>
```

Comment:
```suggestion
      <p>Such dragging movements are difficult or impossible to carry out for some users with motor disabilities. The alternative to dragging movements operates the underlying function by one or several <a>single pointer</a> activations. A single tap or click may reveal controls (arrows) to move a target in a stepwise fashion; open a drop-down menu where the drop position can be selected; or allow moving it to an ajacent postion by a swipe gesture. (Regarding the swipe gesture alternative, note that the swipe gesture alone would not be sufficient since Success Criterion 2.5.1 Pointer Gestures requires that functionality can be operated with a single pointer <strong>without</strong> a path-based gesture, unless a multipoint or path-based gesture is essential.)</p>
```